src/apps/TxBuilder/hooks/useServices/interfaceRepository.ts

```diff
@@ -15,6 +16,24 @@ export interface ContractInterface {
 
 const getContract = memoize(async (apiUrl: string) => axios.get(apiUrl));
 
+const getAbiUrlByNetwork: {
```

Comment:
the functionality is there, I'm not so sure about the `get` part of the name for this map-like object.

Reply:
that's right, I thought it as a function then moved it into an object. any suggestion? 
removing the `get` perhaps?

Reply:
Yes.

In fact, what this returns is not an `abiUrlByNetwork` but a function that returns the `abiUrl`.

So, thinking in that direction I thought something around:
```typescript
const getAbiUrl = abiUrlGetterByNetwork[networkName]

if (getAbiUrl === null) {
  throw Error ('...')
}

const abi = getAbi(getAbiUrl(contractAddress))
```

would be most suitable.

But I'm not the master-on-naming-things :mage_man:, so, your call.

Reply:
done! thx for the detailed feedback :)